public-ui/src/app/data-browser/components/survey-answer/survey-answer-react.component.tsx

```diff
@@ -0,0 +1,255 @@
+
+                    // this.props.SubQuestion.countAnalysis = results.items.filter(a => a.analysisId === 3110)[0];
+                    // console.log(results.questions.items[0].countAnalysis.results);
+                    this.setState({
+                        subQuestion: results.questions.items[0],
+                        subAnswers: results.questions.items[0].countAnalysis.results,
+                        subTitle: results.questions.items[0].conceptName
+                    });
+                    console.log(this.state.subAnswers, 'subAnswers');
+                    // console.log(this.state.subQuestion, 'Sub question');
+
+                });
+    }
+
+    render() {
+        const parcipantPercentage = this.props.countValue / this.props.partcipantCount;
+        return <React.Fragment> <div className='survey-tbl-exp-r survey-tbl-r' onClick={this.openDrawer.bind(this)}>
+            <div className='survey-tbl-d first display-body info-text survey-answer-level-1'>
+                {this.props.stratum4}
+            </div>
+            <div className='survey-tbl-r-group'>
+                <div className='survey-tbl-d display-body info-text survey-answer-level-1'>
+                    {this.props.stratum3}
+                </div>
+                <div className='survey-tbl-d display-body info-text survey-answer-level-1'>
+                    {this.props.countValue.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')}
+                </div>
+                <div className='survey-tbl-d display-body info-text survey-answer-level-1'>
+                    {this.props.countPercent ? this.props.countPercent.toFixed(2) : parcipantPercentage.toFixed(2)}%
+                    </div>
+                <div className='survey-tbl-d display-body info-text survey-answer-level-1'>
+                    {this.hasSubQuestions === '1' ?
+                        <ClrIcon shape='caret' className='survey-row-icon' style={{ color: '#216fb4' }} dir={this.state.drawerOpen ? 'down'
+                            : 'right'} /> : <ClrIcon className='survey-row-icon' shape='bar-chart' />}
+                </div>
+            </div>
+        </div >
+            {this.state.drawerOpen ? <div className='survey-row-expansion'>
+                {(this.hasSubQuestions === '1' && this.state.subQuestion) ? <React.Fragment>
+                    <h6 className='sub-question-text'><ClrIcon shape='child-arrow' />{this.state.subTitle}</h6>
+                    <div className='survey-sub-table'>
+                        {/* tslint:disable-next-line: no-use-before-declare  */}
+                        <SurveyAnswerReactComponent
+                            particpantCount={this.props.countValue}
+                            question={this.state.subQuestion} isCopeSurvey={this.props.isCopeSurvey} />
+                    </div>
+                </React.Fragment> :
+                    <h5>graph-component</h5>
+                }
+            </div> : undefined}
```

Comment:
Goes with the above comment about removing ternary operator:
```suggestion
            </div>}
```